bin/sdmerge

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
+
+set -euo pipefail # STRICT MODE
+IFS=$'\n\t'       # http://redsymbol.net/articles/unofficial-bash-strict-mode/
+
+# Takes two arguments – a GitHub repository name and a PR number
```

Comment:
Update this to describe the new behavior, or better, integrate it into the usage message and move the usage message earlier in the script.